pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "temporalio"
-version = "1.1.0"
+version = "1.2.0"
```

Comment:
Why do a minor instead of a patch?

Reply:
Is it due to behavior changes in Core?

Reply:
In general, for the SDK releases I've found always assuming minor except for emergency hotfixes makes for a clearer stream of releases. Last release was over two months ago, so this can be a standard release IMO (there just aren't enough features to have a regular release cadence). But for the _pure_ definition of semver, this and the other things done in the last two months are technically a patch. If a minor is bothersome enough I can change it.